homeassistant/components/flume/sensor.py

```diff
@@ -0,0 +1,288 @@
+"""Sensor for displaying the number of result from Flume."""
+import base64
+from datetime import datetime, timedelta
+import json
+import logging
+
+import pytz
+import requests
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, CONF_PASSWORD, CONF_USERNAME
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = "Flume Sensor"
+
+CONF_CLIENT_ID = "client_id"
+CONF_CLIENT_SECRET = "client_secret"
+CONF_TIME_ZONE = "time_zone"
+
+SCAN_INTERVAL = timedelta(minutes=1)
+
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Required(CONF_CLIENT_ID): cv.string,
+        vol.Required(CONF_CLIENT_SECRET): cv.string,
+        vol.Required(CONF_TIME_ZONE): cv.string,
```

Comment:
I think we should get the time zone from Home Assistant.

Reply:
Thank you for the feedback! I have updated the sensor accordingly.